ott/core/problems.py

```diff
@@ -321,7 +335,7 @@ def is_balanced(self):
 
   def tree_flatten(self):
     return (
-        [self.geom_1, self.geom_2, self._a, self._b],
+        [self.geom_xx, self.geom_yy, self._a, self._b],
```

Comment:
shouldn't the geom_xy appear here? as well as the fused_penalty and is_fused flags?

Reply:
True, thank you, added that to the follow-up pull request.